images/archlinux/Containerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/library/archlinux:base-devel
+
+ENV NAME=archlinux-toolbox VERSION=base-devel
+LABEL com.github.containers.toolbox="true" \
+      name="$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Arch Linux toolbox containers" \
+      maintainer="Erazem Kokot <contact@erazem.eu>"
+
+COPY README.md /
+
+# Install extra packages
+COPY extra-packages /
+RUN pacman -Syu --needed --noconfirm - < extra-packages
+RUN rm /extra-packages
+
+# Clean up cache
+RUN pacman -Scc --noconfirm
+
+# Enable sudo permission for wheel users
+RUN sed -i 's/# %wheel ALL=(ALL) NOPASSWD: ALL/%wheel ALL=(ALL) NOPASSWD: ALL/' /etc/sudoers
```

Comment:
```suggestion
RUN echo "%wheel ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/toolbox
```

Would be simpler.

Reply:
Thanks for the suggestion :)

Didn't realize Github allowed committing suggestions, I'll use that next time.